packages/gatsby-telemetry/src/telemetry.js

```diff
@@ -185,10 +185,10 @@ module.exports = class AnalyticsTracker {
       nodeVersion: process.version,
       platform: os.platform(),
       release: os.release(),
-      cpus: cpus && cpus.length > 0 && cpus[0].model,
+      cpus: (cpus && cpus.length > 0 && cpus[0].model) || undefined,
       arch: os.arch(),
-      ci: ci.isCI,
-      ciName: (ci.isCI && ci.name) || undefined,
+      ci: ci.isCI || !!process.env.CI,
```

Comment:
are you sure this is necessary?
https://github.com/watson/ci-info/blob/master/index.js#L52-L59

Reply:
nice catch! 
Updating the PR